include/bluetooth/mesh/scheduler_cli.rst

```diff
@@ -0,0 +1,72 @@
+.. _bt_mesh_scheduler_cli_readme:
+
+Scheduler Client
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Scheduler Client model remotely controls the state of a :ref:`bt_mesh_scheduler_srv_readme` model.
+
+Unlike the Scheduler Server model, the Scheduler Client only creates a single model instance in the mesh composition data.
+The Scheduler Client can send messages to both the Scheduler Server and the Scheduler Setup Server, as long as it has the right application keys.
+
+Action scheduling example
+=========================
+
+This is a simple example of how the Scheduler Client can configure the Scheduler Server models managing the light system.
+The Scheduler Server will turn the light on every working day at 8:00 and turn the light off at 16:00 every year whole year.
```

Comment:
I struggled a bit with this sentence, so I tried to rewrite it. See if you find it ok, or if we should work on it more.

Reply:
I also discovered that the heading levels were wrong, but otherwise I think this looks good :)
I pushed my commit, please squash it.